src/components/Loader/Loader.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import loaderStyles from "./loader.module.css";
+
+const Loader = () => {
+  return (
+    <div className={loaderStyles.wrapper}>
+      <div class={loaderStyles.spinner}>
```

Comment:
@majac91 I noticed you have multiple React errors like this class (it should be classname). These errors are on the console as well.
I really recommend you to fix them (and understand why they were errors) as they might cause bugs in the production/build :)